workers/Dockerfile

```diff
@@ -0,0 +1,43 @@
+# Simple generic worker
+
+FROM golang:1.18.4-buster as build
+
+WORKDIR /app
+
+RUN git clone --depth 1 --branch main --single-branch "https://github.com/taskcluster/taskcluster" .
+
+ENV CGO_ENABLED=0
+RUN cd tools/livelog && go build -o /livelog && cd ..
+RUN cd tools/worker-runner && go build -o /start-worker ./cmd/start-worker && cd ..
+RUN cd tools/taskcluster-proxy && go build -o /taskcluster-proxy && cd ..
+RUN cd clients/client-shell && go build -o /taskcluster && cd ../..
+RUN cd workers/generic-worker && \
+  ./build.sh && \
+  mv generic-worker-docker-* /generic-worker-docker && \
+  mv generic-worker-multiuser-* /generic-worker-multiuser && \
+  mv generic-worker-simple-* /generic-worker
+
+
+FROM ubuntu:20.04
```

Comment:
I suspect you can change this to `FROM scratch`, and then just add the root ca authority certs file, and the specific binaries that generic-worker depends on (which is probably a small handful). However, we can do this in a future PR if we like, it is an optimisation only. But it could get us from a several hundred megabyte or even gigabyte docker image to one that is e.g. 20-30MB.

Reply:
ah, good point, thanks! I didn't know that generic worker is self-sufficient and doesn't require much from the OS. 
I think we can keep it as is for now, since this is only intended for local dev purposes. Once we gonna ship this image, we'd need to trim it for sure. Thanks